build.sbt

```diff
@@ -37,10 +41,10 @@ lazy val reactiveZookeeper = (project in file("reactive-zookeeper")).settings(
 lazy val reactiveZookeeperExample = (project in file("example"))
   .settings(noPublishSettings)
   .settings(
-  commonSettings ++ Seq(
+  commonSettings ++ publishSettings ++ Seq(
```

Comment:
@j5ik2o 
Note that "example" project is not expected to be published.
At the line number 42 you can find `settings(noPublishSettings)`.

Reply:
@TanUkkii007 Oops, fixed.